### PR TITLE
Change modelpage to modal type 

### PR DESCRIPTION
**Pull request by Brianballow**
## Changes

1. When thumbnail is clicked, open modal window instead of pushing model page.
2. Keep previous page's search position after exiting modal.

### CODE REVIEW

File: components/Thumbnails.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Image from "next/image";
 import { useRouter } from "next/router";
 import { increaseView } from "pages/models/[id]";
 import { Dispatch, MouseEvent, SetStateAction, useState } from "react";
+import ModelModal from "./ModelModal";
```
Comment (Taehyeong-Gim) [inline components/Thumbnails.tsx:7]: Following previous importing code convention, please use "@components/ModelModal"

**Review by Taehyeong-Gim — COMMENTED** [review]
Here is an proposal for keeping coding convention. Please consider this.

**Comment by softmare** [comment]
> Following previous importing code convention, please use "@components/ModelModal" 

Sure. No problem!

File: components/Thumbnails.tsx (hunk 1 above)
Comment (Brianballow) [reply components/Thumbnails.tsx:7]: Sure, No problem!

**Pull request merged**